src/citra_qt/main.ui

```diff
@@ -188,6 +200,25 @@
     <string>Create Pica Surface Viewer</string>
    </property>
   </action>
+  <action name="action_Start_Room">
+   <property name="enabled">
+    <bool>true</bool>
+   </property>
+   <property name="text">
+    <string>Create/Destory Room</string>
```

Comment:
Only needs to be "Create new Room" [also spelling in Destory which is to be removed ;)  ]